libs/cart/driver/magento/testing/src/factories/cart-item/cart-item.factory.ts

```diff
@@ -19,6 +19,7 @@ export class MockMagentoCartItem implements MagentoCartItem {
     price: this.money(),
     row_total: this.money(),
     row_total_including_tax: this.money(),
+    discounts: this.discounts(faker.random.number({min: 1, max: 5})),
```

Comment:
I think this should have a  minimum of 0

Reply:
changed